src/avram/database.cr

```diff
@@ -153,7 +153,9 @@ abstract class Avram::Database
 
   # :nodoc:
   def run
-    yield current_connection || db
+    yield current_connection
```

Comment:
The `|| db` was unnecessary as `current_connection` checks out a connection if there isn't already one